controls/cf_serverd.cf

```diff
@@ -32,9 +32,6 @@ body server control
       # Suggested value >= total number of hosts
       maxconnections        => "$(def.control_server_maxconnections)";
 
-      # Bind to all interfaces including ipv6
```

Comment:
@olehermanse I think @nickanderson 's suggestion will have the same effect with what you did in this patch.